tests/hpc/slurm_config.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017-2018 SUSE LLC
```

Comment:
please only use the lines during which you worked on the file, i.e. 2019 I assume :)

Reply:
copy&paste syndrome of course. I'll correct it then